packages/desktop-client/src/components/settings/Format.js

```diff
@@ -98,6 +115,17 @@ export default function FormatSettings({ prefs, savePrefs }) {
               />
             </Button>
           </Column>
+
+          <Column title="First day of the week">
+            <Button bounce={false} style={{ padding: 0 }}>
+              <CustomSelect
+                value={firstDayOfWeekIdx}
+                onChange={onFirstDayOfWeek}
+                options={daysOfWeek.map(f => [f.value, f.label])}
```

Comment:
An alternative, which would make it easier to localize in the future (although 

```js
import locale from 'date-fns/locale/en-US';
```

```suggestion
                options={Array(7).fill().map((_, i) => [String(i), locale.localize.day(i)])}
```

Or:

```suggestion
                options={daysOfWeek.map((day, i) => [String(i), day])}
```

and then:

```js
let daysOfWeek = [
  'Sunday',
  'Monday',
  'Tuesday',
  'Wednesday',
  'Thursday',
  'Friday',
  'Saturday',
];
```